index.html

```diff
@@ -0,0 +1,119 @@
+<body>
+    <nav class="navbar">
+			<div class="container-center">
+				<div class="hamburger-menu pointer" role="button" tabindex="0">
+					<span class="hamburger-menu-line"></span>
+					<span class="hamburger-menu-line"></span>
+					<span class="hamburger-menu-line"></span>
+                </div>
+
+				<a href="/" class="container-center brand-logo">
+					<p class="brand-name">Nature <span>UI</span></p>
+				</a>
+			</div>
+			<div class="container-center">
+				<a href="/" class="flex">
+					<!-- <img src="./nature.png" alt="avatar" class = "circle-border avatar-img sm"> -->
+					Github
+					<i
+						style="font-size: 1.5rem; margin-left: 0.2rem"
+						class="fa fa-github"
+						aria-hidden="true"
+					></i>
+				</a>
+			</div>
+
+			<ul class="menu hide-menu" id="menu">
+				<li class="menu-item">
+					<b><a href="index.html">Home</a></b>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html">Installation</a>
+				</li>
+				<li class="menu-item">
+					<b><a href="./components.html#alerts">Components</a></b>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#alerts">Alerts</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#buttons">Buttons</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#avators">Avatar</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#listgroups">List Groups</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#badges">Badges</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#productcards">Product Card</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#textcards">Text Card</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#progress">Progress</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#forms">Forms</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html" aria-disabled="true">Examples</a>
+				</li>
+				<li class="hide-in-side-nav menu-item">
+					<a href="/" class="flex">
+						<!-- <img src="./nature.png" alt="avatar" class = "circle-border avatar-img sm"> -->
+						Github
+						<i
+							style="font-size: 1.5rem; margin-left: 0.2rem"
+							class="fa fa-github"
+							aria-hidden="true"
+						></i>
+					</a>
+				</li>
+			</ul>
+		</nav>
+		<div style="padding: 3rem; padding-top: 6rem; display: flex; flex-direction: column; justify-content: center;">
+			<h1 style="font-size: 4rem;">
+				Build fast, light weight sites with Nature UI.
+			</h1><br>
+			<p style="font-size: 1.5rem; padding-left: 0.5rem; line-height: 2.5rem; color: #4b5563;">
+				Nature-UI is a simple and accessible component library that gives you the building blocks you need to build your applications.
+			</p><br><br>
+			<div >
+				<a href="./components.html"><button class="btn btn-primary" style=" font-size: 1.5rem;"><b>Get Started</b></button></a> &nbsp; &nbsp;
```

Comment:
try not to use Html characters for spacing, use padding instead

Reply:
Done with changes🙌🏻.